suites/quincy/cephadm/tier-0_cephadm.yaml

```diff
@@ -0,0 +1,814 @@
+#===============================================================================================
```

Comment:
Could we avoid using `tier-0`? This does not match with the Polarion tier levels. 

Instead opt to name the file that is relevant to what is being tested. Example
`tier-1_dfg-test-cephadm-regression`.

We could also split them into multiple suites that focus on a specific command or workflow